.github/workflows/nightly_ondemand.yml

```diff
@@ -171,6 +171,7 @@ jobs:
       - name: Show GITHUB_ENV
         run: |
           echo "$GITHUB_ENV"
+          rm -rf ${{ github.workspace }}/summary_accuracy.log
```

Comment:
Seems the path not align with e2e action, is it expected? If they are two different files, please use different file name